src/utils/_colors.scss

```diff
@@ -12,25 +12,25 @@
 // $c-primary-dark: color-variation($c-primary, -2) -> 2 steps darker
 // $c-primary-darkest: color-variation($c-primary, -4) -> 4 steps darker
 
+$step-variations-positive: 25, 50, 75, 90;
```

Comment:
this should have !default at the end, and maybe have a more descriptive name like color-darken-steps and color-lighten-steps

Reply:
Ok 👍